### DISCUSSION TITLE
Represent golf courses and their features

### DISCUSSION THREAD

**Discussion by patepelo** [organicmaps/organicmaps#6661] [Ideas]
Location for the examples: https://www.openstreetmap.org/#map=15/49.2455/8.6237

Here is the current carto representation:
![image](https://github.com/organicmaps/organicmaps/assets/23003127/298070f2-9377-4064-a025-92daa1f953ae)

Here is the current OM representation of the same track.
![image](https://github.com/organicmaps/organicmaps/assets/23003127/8d5f7398-adae-4ff4-a3fe-e2ba339aa908)

Some tags missing:

golf=bunker
natural=sand
___
golf=green
___
golf=fairway
natural=grassland
___
golf=pin

___
golf=tee
natural=grassland
___
golf=water_hazard
natural=water
water=lake
____

See [wiki leisure=golf_course](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dgolf_course)

**Comment by hb0nd** [comment]
another example https://www.openstreetmap.org/way/386763895#map=18/51.49866/-0.00018

![image1](https://github.com/organicmaps/organicmaps/assets/26939824/4d55614a-d4db-4b91-b70b-3148160c5708)

**Comment by platlas** [comment]
Hope this is not off-topic but would be nice if tags `disc_golf=hole`, `disc_golf=tee` and `disc_golf=basket` were included as well.  https://wiki.openstreetmap.org/wiki/Tag:leisure%3Ddisc_golf_course
